test/cypress/tests/copy-paste.spec.ts

```diff
@@ -1,13 +1,48 @@
 import Header from '@editorjs/header';
 import Image from '@editorjs/simple-image';
 import * as _ from '../../../src/components/utils';
+import { BlockTool, BlockToolData } from '../../../types';
+import $ from '../../../src/components/dom';
 
 describe('Copy pasting from Editor', function () {
+  const onPasteStub = {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onpaste: () => {},
+  };
+
+  /**
+   * Tool with disabled preventing default behavior of onPaste event
+   */
+  class BlockToolWithPasteHandler implements BlockTool {
+    public static pasteConfig = false;
+
+    /**
+     * Render block
+     */
+    public render(): HTMLElement {
+      const block = $.make('div', 'ce-block-with-disabled-prevent-default', {
+        contentEditable: 'true',
+      });
+
+      block.addEventListener('paste', onPasteStub.onpaste);
+
+      return block;
+    }
+
+    /**
+     * Save data method
+     */
+    public save(): BlockToolData {
+      return {};
+    }
+  }
+
   beforeEach(function () {
     cy.createEditor({
       tools: {
         header: Header,
         image: Image,
+        blockToolWithPasteHandler: BlockToolWithPasteHandler,
```

Comment:
I think it would be better to create an instance with this tool inside the corresponding test, not before each test case since it is not related to other tests.